etna/lib/etna/clients/magma/workflows/record_synchronization_workflow.rb

```diff
@@ -0,0 +1,57 @@
+require 'ostruct'
+
+module Etna
+  module Clients
+    class Magma
+      class RecordSynchronizationWorkflow < Struct.new(:target_client, :source_client, :project_name, keyword_init: true)
```

Comment:
Starting place for copying magma records between environments.  Kind of simple at the moment, doesn't handle tables correctly currently.  I'd like to pair this with a smart fuzzier to increase safety against PHI in a future PR.